components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -96,11 +96,12 @@ end
 function AutomaticPointsTable:parseInput(args)
 	local positionBackgrounds = self:parsePositionBackgroundData(args)
 	local tournaments = self:parseTournaments(args)
-	local teams = self:parseTeams(args, #tournaments)
+	local resolveRedirect = Logic.readBool(args.resolveRedirect)

```

Comment:
```suggestion
	local shouldResolveRedirect = Logic.readBool(args.resolveRedirect)
```
And rename everywhere else too. Makes it clearer this is a bool, and what the bool represents.